contrib/check-config.sh

```diff
@@ -182,8 +185,12 @@ echo 'Optional Features:'
 		echo "    $(wrap_color '(note that cgroup swap accounting is not enabled in your kernel config, you can enable it by setting boot option "swapaccount=1")' bold black)"
 	fi
 }
+
+if (( $kernel_version_major <= 3 && $kernel_version_minor <= 18 )); then
```

Comment:
This syntax looks cool, but it's inconsistent with the rest of our scripts (and I'm not sure how broadly-supported it is).
```bash
if [ "$kernelMajor" -le 3 -a "$kernelMinor" -le 18 ]; then
```

Also, what does this mean for `2.19+` kernels (like RedHat)?  Should this really be something more like the following, technically?
```bash
if [ "$kernelMajor" -lt 3 ] || [ "$kernelMajor" -eq 3 -a "$kernelMinor" -le 18 ]; then
```